### DISCUSSION TITLE
Discord Rich Presence

### DISCUSSION THREAD

**Discussion by Peytoncl** [nukeop/nuclear#826] [Ideas]
You should add "Discord Rich Presence" (if thats possible with electron) :)

**Comment by nukeop** [comment]
We already have that: https://github.com/nukeop/nuclear/blob/d0b384789bd1eb3b8d5a136a4aa1dd46c4db3e97/packages/main/src/services/discord/index.ts

**Comment by Peytoncl** [comment]
> We already have that: https://github.com/nukeop/nuclear/blob/d0b384789bd1eb3b8d5a136a4aa1dd46c4db3e97/packages/main/src/services/discord/index.ts

no way 😮 my bad 😅
